client/src/app/shared/shared-account-avatar/account-avatar.component.html

```diff
@@ -1,5 +1,8 @@
 <ng-template #img>
-  <img [class]="class" [src]="avatarUrl" i18n-alt alt="Account avatar" />
+  <img *ngIf="avatarUrl || !initial" [class]="class" [src]="avatarUrl || defaultAvatarUrl" i18n-alt alt="Account avatar" />
```

Comment:
Why would we still need the default avatar url?

Reply:
I saw in some places where the avatar was used that it was conditional checks if the `account` was set, so I assumed there's cases when that's applicable. For example here https://github.com/Chocobozzz/PeerTube/blob/e771e82dfac5f8cd5462fecf6618f9b1fb314e30/client/src/app/%2Bvideos/%2Bvideo-watch/comment/video-comment.component.ts#L108

Can it be if a an user/account has been deleted but the comments still exists?